dart/lib/flat_buffers.dart

```diff
@@ -142,13 +144,17 @@ class Builder {
   /// automatically grow the array if/as needed.  `internStrings`, if set to
   /// true, will cause [writeString] to pool strings in the buffer so that
   /// identical strings will always use the same offset in tables.
-  Builder({this.initialSize: 1024, bool internStrings = false}) {
+  Builder(
+      {this.initialSize: 1024, bool internStrings = false, Allocator allocator})
+      : _allocator = allocator ?? DefaultAllocator() {
     if (internStrings == true) {
       _strings = new Map<String, int>();
     }
     reset();
   }
 
+  int get size => _tail + ((-_tail) % _maxAlign);
```

Comment:
making this a getter now invokes a clsoure and does the math every time we access this. It should be possible to just keep it calculated up front right? IF the allocator impl is affecting this, just make it be on there, and the default one manages like it is currently.

Reply:
It isn't related to the allocator implementation. I'll extract a separate PR.

Reply:
OK, it is kinda related but I've still pushed a separate PR #6403